Test.Utility/MockModuleBuilder.cs

```diff
@@ -1,51 +1,16 @@
-﻿using System;
-using System.Collections.Generic;
-using Autofac;
-using Autofac.Core;
+﻿using Messerli.CompositionRoot;
 using Moq;
 
 namespace Messerli.Test.Utility
 {
-    public class MockModuleBuilder
+    public static class MockModuleBuilder
```

Comment:
Rename class to something with ´Extension´ in the name i. E ´ModuleBuilderMockExtension´

Reply:
Addressed in: https://github.com/messerli-informatik-ag/test-utility/pull/20/commits/4d33a0c474230aebf9a54c1d4d870ecb7c356783